src/pages/trading-specification/index.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import styled from 'styled-components'
+import Layout from 'components/layout/layout'
+import { SectionContainer, SEO } from 'components/containers'
+import { Header } from 'components/elements'
+import MarketTab from 'components/elements/market-tab'
+import { Localize, localize, WithIntl } from 'components/localization'
+import device from 'themes/device'
+
+const StyledHeader = styled(Header)`
+    text-align: center;
+    font-size: 48px;
+
+    @media ${device.tablet} {
+        font-size: 28px;
+        padding: 0 41px;
+    }
+`
+const TradingSpecification = () => {
+    return (
+        <Layout>
+            <SEO
+                title={localize('Trading specifications | CFD trading | Trading with Deriv')}
+                description={localize(
+                    'Find Deriv’s CFD trading specifications for multiple assets in forex, derived indices, stocks and indices, cryptocurrencies, and commodities.',
+                )}
```

Comment:
Exclusively for SEO component we can write `localize` function
```suggestion
                title={localize('_t_Trading specifications | CFD trading | Trading with Deriv_t_')}
                description={localize(
                    '_t_Find Deriv’s CFD trading specifications for multiple assets in forex, derived indices, stocks and indices, cryptocurrencies, and commodities._t_',
                )}
```

Reply:
Hi @NikitK-deriv, fixed all these issues in new commit. Thanks :)